README.md

```diff
@@ -16,6 +16,17 @@ Deis v2 and Helm are changing quickly. Your feedback and participation are more
 
 Hacking on Deis v2 is a bit rough right now--please help us fix the bugs and improve the installation process.
 
+### Prerequisites
+You must enable the extensions api for your kube-apiserver. For now the only requirement is that the daemonsets portion of the api be enabled. To do this add the following line to the script that starts your api server:
+
+```console
+--runtime-config=extensions/v1beta1=true,extensions/v1beta1/daemonsets=true
+```
+
+you can also do `ENABLE_DAEMONSETS=true` before running the `kube-up.sh` script.
+
+### Installation 
+
 First, add this Chart repo to Helm to install the "deis" chart:
```

Comment:
yeah its hard to know all the possible ways that someone will provison k8s. I added the only two ways I know that you could enable daemonsets api. However, right now there is no reason to include this section in the main `deis` chart readme. Instead it should be put in the `deis-logger` chart readme.